src/scripts/main.js

```diff
@@ -1,3 +1,217 @@
 'use strict';
 
 // write your code here
+const button = document.querySelector('.button');
+const messeageStart = document.querySelector('.message-start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const gameScore = document.querySelector('.game-score');
+const rows = document.querySelectorAll('.field-row');
+let score = 0;
+let gameField = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
+
+function setGameField() {
+  for (let row = 0; row < 4; row++) {
+    for (let cell = 0; cell < 4; cell++) {
```

Comment:
Why do you call it cell? It's more logical to call it column as it represents the column. You should change it all over the project.